src/pages/Profile/index.js

```diff
@@ -23,90 +32,361 @@ import avatar from '../../assets/avatar.svg';
 class Profile extends React.Component {
   constructor(props) {
     super(props);
+    this.ref = {
+      name: React.createRef(),
+      email: React.createRef(),
+      position: React.createRef(),
+    };
     this.state = {
-      isChecked: true,
+      dataExpireValue: '6 months',
+      editView: false,
+      displayName: 'Anisha Swain',
+      emailVal: 'xyz.redhat@gmail.com',
+      positionVal: 'Associate Software Engineer',
+      accountType: 'Pbench',
+      accountStatus: 'Active',
+      color: '#0080005e',
     };
-  }
 
-  componentDidMount = () => {};
+    this.dataExpireOptions = [
+      { value: '2 months', label: '2 months', disabled: false },
+      { value: '6 months', label: '6 months', disabled: false },
+      { value: '1 year', label: '1 year', disabled: false },
+    ];
+
+    this.accountOptions = [
+      { value: 'Pbench', label: 'Pbench', disabled: false },
+      { value: 'Google', label: 'Google', disabled: false },
+      { value: 'RedHat SSO', label: 'RedHat SSO', disabled: false },
+      { value: 'Github', label: 'Github', disabled: false },
```

Comment:
we can use a `mock` file for that

Reply:
The `accountOptions` array is just for showing the values in the dropdown in a proper format. Plus I don't think we are going to store all the options as user data in the backend. Are we!

However, the type of the user account needs to be stored in backend, so it is set in the component state which is grabbed from a mock user file(which will be replaced with data from an API request later)

Reply:
My assumption is that we use SSO to authenticate a user and provide initial registration information, but subsequently they'd log in using a Pbench username/email and password. In that case, it may be that the backend wouldn't need to know. (Depending on whether we want other front end tools to be able to perform the SSO "registration" through the backend APIs directly.)

However if the idea is that users could continue to log in via SSO rather than creating a local server Pbench password, the backend would need to understand the SSO protocols and certainly would need to know which to use for each user.